### PR TITLE
Admin can edit users

### PR DESCRIPTION
**Pull request by emanuel-metzenthin**
Issue #189

### CODE REVIEW

**Comment by emanuel-metzenthin** [comment]
The code climate error doesn't make sense in my opinion because the password has to be deleted from the params and any extraction into variables or methods would only make the code less readable. (@jak-ing)

**Review by jonashering — CHANGES_REQUESTED** [review]
Regarding the authorization management for admin, this looks very good to me!

Just one small request: Clicking on My Account in the Navbar currently results in an error (links to something like `http://localhost:3000/my/users/edit.1`). Maybe changing the link in `_navbar.html.erb` from `edit_user_registration_path`to `edit_user_path` as you did in some of the other files could solve this. Thanks!

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -17,6 +17,35 @@ def show
...
+  def edit
+    @user = User.find(params[:id])
+    authorize! :edit, @user
+  end
+
+  def update
+    @user = User.find(params[:id])
+    authorize! :update, @user
+
+    unless user.admin?
```
Comment (chrisma) [inline app/controllers/users_controller.rb]: @user?

Comment (MariusDanner) [reply app/controllers/users_controller.rb]: What do you mean with that?

Comment (chrisma) [reply app/controllers/users_controller.rb]: I was wondering whether line 29 should read `unless @user.admin?`

Comment (MariusDanner) [reply app/controllers/users_controller.rb]: Ah ok, I see.
It is supposed to be current_user I think

**Review by jonashering — APPROVED** [review]

**Pull request merged**